code/modules/mob/living/simple_animal/hostile/bees.dm

```diff
@@ -32,7 +32,7 @@
 	environment_smash = 0
 	mouse_opacity = 2
 	pass_flags = PASSTABLE | PASSGRILLE | PASSMOB
-	mob_size = MOB_SIZE_SMALL
+	mob_size = MOB_SIZE_TINY
```

Comment:
While your here could you remove their mouse opacity, it was only set because the sprite used to be a massive swarm of pixel sized bees with moving voids of "you didn't actually click ON the mob!"

Now that it's just one bee with a well defined center of mass all the opacity does is make it a pain to collect veggies under bees